web/src/javascript/fakeState.js

```diff
@@ -37,7 +37,7 @@ export default {
       {
         "id": 13,
         "name": "wow",
-        "total_mass": 555,
+        "total_mass": 455,
```

Comment:
I'm just curious.. Is the total_mass the score that will appear for the game (meaning the score done since the player is alive) or is it the sum of all the player's cells mass?